app/proguard-rules.pro

```diff
@@ -78,6 +78,7 @@
 -keep class io.intercom.android.** { *; }
 -keep class com.intercom.** { *; }
 -keep class com.appsflyer.** { *; }
+-keep class kotlin.coroutines.intrinsics.** { *; }
```

Comment:
the obfuscation problem 
<img width="1009" alt="2023-01-19_17 48 01" src="https://user-images.githubusercontent.com/121934748/213505153-1703f505-d78b-403d-8188-b7f3cb4e980f.png">